0-1BFS/0-1_BFS.cpp

```diff
@@ -0,0 +1,113 @@
+ /* Given a graph where every edge has weight as either 0 or 1. 
+ A source vertex is also given in the graph. 
+ Find the shortest path from source vertex to every other vertex.
+ Complexity O(V+E) */
+
+#include<bits/stdc++.h> 
+#define V 1000
+
+using namespace std; 
+
+int dist[V];  // array to store the shortest distances
+// the graph stores vectors of int pairs such that graph[node].first = destination node & graph[node].second = edge weight
+vector < pair <int , int> > graph[V];
+deque <int> Q; // keeps track of edges during BFS
+
+//FUNCTIONS: 
+
+int distbfs(int i, int v) // returns the new distance to the destination node
+{
+    return dist[v] + graph[v][i].second;
+}
+
+/* To print shortest distance from given source (here, 0) to 
+ every other node */
+void zeroOneBFS(int source,int n) 
+{ 
+    // Marking distances of all nodes from the source except itself as infinity  
+    int i = 0;
+    while (i < n) 
+    {
+	if (i == source)
+	{
+		dist[i] = 0; // distance of source from itself is 0
```

Comment:
Please use 4 space indentation throughout the file.

Reply:
Commit #7eb6633

Reply:
Can you pls help me about what changes to make?